internal/pkg/cli/app_init.go

```diff
@@ -22,6 +22,19 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	appInitAppTypePrompt     = "Which type of " + color.Emphasize("infrastructure pattern") + " best represents your application?"
+	appInitAppTypeHelpPrompt = `Your application's architecture. Most applications need additional AWS resources to run.
+To help setup the infrastructure resources, select what "kind" or "type" of application you want to build.`
+
+	fmtAppInitAppNamePrompt     = "What do you want to " + color.Emphasize("name") + " this %s?"
+	fmtAppInitAppNameHelpPrompt = `The name will uniquely identify this application within your %s project.
+Deployed resources (such as your service, logs) will contain this app's name and be tagged with it.`
+
+	fmtAppInitDockerfilePrompt  = "Which Dockerfile would you like to use for %s app?"
```

Comment:
Would it be better to put like `Which Dockerfile would you like to use for %s?`

Reply:
Done!